tests/end_to_end/test_vulnerability_detector/cases/test_vulnerability.yaml

```diff
@@ -104,13 +100,11 @@
           state_index: true
         package:
           centos:
-            amd64: grafana-8.5.5
-            arm64v8: grafana-8.5.5
+            amd64: openjdk-1.6.0"
```

Comment:
```suggestion
            amd64: openjdk-1.6.0
```
Those quotation marks are causing an error in the tests, as can be seen in the Linux test report:

```
> package_data = load_packages_metadata()[package_id]
E KeyError: 'openjdk-1.6.0"'
```